github authoring-docs (PR from TinaCMS) (#3675)
tinacms/tina.io: content/docs/tina-cloud/git-co-authoring.mdx

To solve this, TinaCMS provides two distinct methods for attributing your work: **Act as You** (the recommended approach) or **Co-authoring**.

***

### Authoring Options at a Glance

You can choose between two levels of GitHub integration:

1. **Act as You (Recommended):** The most seamless and accurate method. TinaCMS will perform all Git actions using **your personal GitHub identity** after you grant authorization.
2. **Co-authoring:** A simpler, manual option. The `tina-cloud-app` will still make the commit, but it will add your name and email as a **co-author**, attributing the change to you in the commit message.

***

## Option 1: Act as You (Full GitHub Authoring)

This method provides perfect attribution by using your GitHub account to perform all actions. Commits, branches, and pull requests will appear in your repository's history exactly as if you had made them from your own computer.

![A commit authored solely by the user, making it clear who made the change.](/img/docs/tina-cloud/github-authoring/github-authoring-commit-example.png "The commit and PR is created by the user, not the bot.")

### How Permissions Are Secured 🔐

> This feature operates on a **dual-permission system**, meaning an action is only successful if **both** the TinaCMS application and **your personal user account** have the required permissions.

Reviewer: ```suggestion
This feature operates on a **dual-permission system**, meaning an action is only successful if **either** **your personal user account** has the required permissions or the TinaCMS application does.
```
Isn't it an either/or model?

We use personal permission by default (when enabled) then fallback to the bot if they don't have access

5. Save your settings.

![The user settings UI showing the input fields for the Git co-authoring settings.](/img/docs/tina-cloud/github-authoring/co-authoring-input-fields.png "Co-author details are manually entered via the Git Co-authoring profile settings.")

> **Note on Email Privacy:** The co-author email is publicly viewable on GitHub. If you wish to keep your email private, GitHub provides a `noreply` email address you can use. You can find this in your [GitHub email settings](https://github.com/settings/emails).

***

### Quick Comparison

| Feature                    | Act as You (Recommended)                                                                      | Co-authoring                                                                   |
| -------------------------- | --------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
| **Attribution Method**     | Commits are made directly by your user identity.                                              | Commits are made by the bot, with your name added as a `Co-authored-by:` line. |

> | **Security & Permissions** | Secure dual-permission model. Actions are limited by both the app's and your own permissions. | No extra permissions required. Relies only on the bot's permissions.           |

Reviewer: given the above comments, this may need updating
